react/defaults.js

```diff
@@ -0,0 +1,149 @@
+import PropTypes from 'prop-types'
+
+export const lean = {
+  propTypes: {
+    /** If it is leanMode or not */
+    leanMode: PropTypes.bool,
+  },
+  defaultProps: {
+    leanMode: false,
+  },
+}
+
+export const logo = {
+  propTypes: {
+    /** Address opened when the user clicks the logo */
+    linkUrl: PropTypes.string,
+    /** URL of the logo image */
+    logoUrl: PropTypes.string,
+    /** Alt text for the logo */
+    logoTitle: PropTypes.string,
+    /** Size of the logo */
+    logoSize: PropTypes.shape({
+      /** Size on desktop */
+      desktop: PropTypes.shape({
+        width: PropTypes.number,
+        height: PropTypes.number,
+      }),
+      /** Size on mobile */
+      mobile: PropTypes.shape({
+        width: PropTypes.number,
+        height: PropTypes.number,
+      }),
+    }),
+  },
+  defaultProps: {
+    linkUrl: '/',
+    logoUrl: '',
+    logoTitle: '',
+    logoSize: {
+      desktop: {
+        width: 132,
+        height: 40,
+      },
+      mobile: {
+        width: 90,
+        height: 40,
+      },
+    },
+  },
+}
+
+export const searchBar = {
+  propTypes: {
+    /** Sets whether the search bar is visible or not */
+    showSearchBar: PropTypes.bool,
+  },
+  defaultProps: {
+    showSearchBar: true,
+  },
+}
+
+export const login = {
+  propTypes: {
+    /** Sets whether the login button is displayed or not*/
+    showLogin: PropTypes.bool,
+  },
+  defaultProps: {
+    showLogin: true,
+  },
+}
+
+export const icons = {
+  propTypes: {
+    /** Classes for icons */
+    iconClasses: PropTypes.string,
+    /** Classes for labels */
+    labelClasses: PropTypes.string,
+  },
+  defaultProps: {
+    iconClasses: 'c-on-base',
+    labelClasses: 'c-on-base',
+  },
+}
+
+export const collapsible = {
+  propTypes: {
+    collapsibleAnimation: PropTypes.shape({
+      /** If should animate on scroll */
+      onScroll: PropTypes.bool,
+      /** If should animate on every scroll up or  down */
+      always: PropTypes.bool,
+      /** Scroll value that animation starts to be active */
+      anchor: PropTypes.number,
+      /** Initial height */
+      from: PropTypes.number,
+      /** Target height */
+      to: PropTypes.number,
+      /** Animation configuration presets: know more @ https://www.react-spring.io/docs/hooks/api */
+      preset: PropTypes.oneOf([
+        'default',
+        'gentle',
+        'wobbly',
+        'stiff',
+        'slow',
+        'molasses',
+      ]),
+      /** Animation configuration: know more @ https://www.react-spring.io/docs/hooks/api */
+      config: PropTypes.shape({
+        mass: PropTypes.number,
```

Comment:
You can put docs here too, to explain to each these props are.

Reply:
I think it's already well documented on [react-spring docs](http://react-spring.surge.sh/#/), that's why the link is provided.